device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubStatusCode.java

```diff
@@ -210,6 +210,8 @@ public static boolean isRetryable(IotHubStatusCode statusCode)
             case THROTTLED:
             case INTERNAL_SERVER_ERROR:
             case SERVER_BUSY:
+            case IO_ERROR:
+            case DEVICE_OPERATION_TIMED_OUT:
```

Comment:
In this function that labels these cases as retryable or not, these two cases were accidentally left out